.NET/Samples/SimpleConsole/packages.config

```diff
@@ -1,4 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net461" />
+  <package id="NuGet.CommandLine" version="4.3.0" targetFramework="net462" developmentDependency="true" />
```

Comment:
Perhaps we should remove this as not really required here. Devs reading the sample may get the wrong impression.

Reply:
Ok. I will remove

Reply:
If I remove this, then I can not pass the build because of the missing Nuget.CommandLine

Reply:
Even this System.Collections.Immutable may not be downloaded. Maybe we create a tool/package project to reference these packages.